modules/apps/remote-app/remote-app-service/src/main/resources/META-INF/portlet-model-hints.xml

```diff
@@ -19,6 +19,9 @@
 		<field name="customElementURLs" type="String">
 			<hint-collection name="CLOB" />
 		</field>
+		<field localized="true" name="description" type="String">
+			<hint-collection name="TEXTAREA" />
```

Comment:
Isn't this `CLOB` instead of `TEXTAREA`? :thinking: 

IIRC, there are only `CLOB` and `TEXT` in SQL.

Reply:
I've checked and there are some more, as TEXTAREA and URL: https://github.com/liferay/liferay-portal/blob/master/portal-impl/src/META-INF/portal-model-hints.xml#L23